extension/resources/walkthrough/live-plots.md

```diff
@@ -0,0 +1,39 @@
+# Monitor your Experiments
+
+As you run your experiments you can use the same
+[`Plots Dashboard`](command:onCommand:dvc.showPlots) to watch for the metrics
+and plot updates:
+
+<p align="center">
+  <img src="images/live-plots-updates.gif"
+       alt="Live Metric Updates" />
+</p>
+
+This functionality comes out of the box. Just keep updating plot files in your
+code, for example like this, using the [DVCLive](https://dvc.org/doc/dvclive)
```

Comment:
This functionality comes baked in when you update your plots files in your training code